notebooks/GoPro2CropKMZ.ipynb

```diff
@@ -0,0 +1,897 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jJYsgRYq9tQR"
+      },
+      "source": [
+        "# GoPro Photos to Crop KMZ\n",
+        "\n",
+        "<a target=\"_blank\" href=\"https://colab.research.google.com/github/nasaharvest/street2sat/blob/gopro2crop/notebooks/GoPro2CropKMZ.ipynb\">\n",
+        "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+        "</a>\n",
+        "\n",
+        "**Author**: Ivan Zvonkov\n",
+        "\n",
+        "**Last Modified**: May 20, 2024\n",
+        "\n",
+        "**Description**: Converts GoPro photos to crop type points. Specifically the notebook:\n",
+        "\n",
+        "1. Downloads GoPro photos from street2sat bucket.\n",
+        "2. Create a dataframe from photos.\n",
+        "3. Extract dates and coordinates.\n",
+        "4. Classify as crop or not crop.\n",
+        "5. Segment crops in crop photos.\n",
+        "6. Filter out low confidence predictions.\n",
+        "7. Move coordinate to field\n",
+        "8. Get Admin Zones for each point\n",
+        "9. Create KMZ file\n",
+        "10. Save the notebook\n",
+        "11. Download KMZ files\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1vsz6lpeBDfO"
+      },
+      "source": [
+        "## Important Prerequisite\n",
+        "\n",
+        "Before running any cell in the notebook,\n",
+        "1. Click the drop down triangle on the top right hand side and select \"Change Runtime Type\".\n",
+        "2. Click the T4 radio button under Hardware Accelerator and click save.\n",
+        "\n",
+        "This will allow the CropSegmentation model to run much faster."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "HooUu8spC4an"
+      },
+      "outputs": [],
+      "source": [
+        "# Required packages\n",
+        "!pip install exifread utm simplekml earthengine-api geemap -q"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bInVS3VT96L3"
+      },
+      "source": [
+        "## 1. Download GoPro photos"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "B4ivKDDG9wzm"
+      },
+      "outputs": [],
+      "source": [
+        "# Login to Google Cloud\n",
```

Comment:
You are now logged in as [[anjalipaliyam@gmail.com](mailto:anjalipaliyam@gmail.com)].
Your current project is [None].  You can change this setting by running:
  $ gcloud config set project PROJECT_ID
  
I am not too familiar with google cloud, so I was a bit confused at this step. Do I just create a new google cloud project and how would I find the ID?